### ISSUE TITLE
Duplicated terminfo profile clashes with system's one

### ISSUE THREAD

**Issue by baco** [barnumbirr/alacritty-debian#4]
Trying to install the package on either Testing or Sid gives the following error:

```
Preparing to unpack .../alacritty_0.8.0-3_amd64_debian_unstable.deb ...
Unpacking alacritty (0.8.0-3) over (0.7.2-2) ...
dpkg: error processing archive /tmp/alacritty_0.8.0-3_amd64_debian_unstable.deb (--unpack):
 trying to overwrite '/usr/share/terminfo/a/alacritty', which is also in package ncurses-term 6.2+20201114-2
Errors were encountered while processing:
 /tmp/alacritty_0.8.0-3_amd64_debian_unstable.deb
```

The problem:
https://packages.debian.org/search?suite=sid&searchon=contents&keywords=%2Fusr%2Fshare%2Fterminfo%2Fa%2Falacritty
| File | Packages |
| --- | --- |
| /usr/share/terminfo/a/alacritty | ncurses-term |

https://packages.debian.org/search?suite=bullseye&searchon=contents&keywords=%2Fusr%2Fshare%2Fterminfo%2Fa%2Falacritty
| File | Packages |
| --- | --- |
| /usr/share/terminfo/a/alacritty | ncurses-term |

The file already exists in a standard package (ncurses-term) in both flavors of the distro

**Comment by barnumbirr** [comment]
Ba**ls, didn't know this was a thing. As @bcbnz is the original reporter of the missing terminfo issue, I'll include him in the discussions. What do you think is the best solution here? Recommend the installation of `ncurses-term` **or** rename `alacritty-debian`'s terminfo file?

**Comment by baco** [comment]
In my opinion, as `ncurses-term` is a `standard` package, meaning it will come in a base Debian installation (packages of priorities: `required`, `important` and `standard` come on a basic Debian Installation when “standard system utilities” task is selected), and as of Debian testing the package `ncurses-term` includes a terminfo for Alacritty; I would keep the file in the `.deb` of the stable version as is and, as suggested, add a recommendation towards `ncurses-term` for the testing and unstable `.deb` packages.

**Comment by barnumbirr** [comment]
This would be quite complicated to manage in the `control` file.
What we could do is not install the terminfo file for buster and let the user take care of that manually while recommending the installation of `ncurses-term` in Bullseye (when it becomes stable), Testing (Bookworm) and Sid.

**Comment by barnumbirr** [comment]
Should be fixed in [v0.9.0-2](https://github.com/barnumbirr/alacritty-debian/releases/tag/v0.9.0-2).
Don't hesitate to reopen this issue if needed.
